release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1341,12 +1341,14 @@ For any {product-title} release, always review the instructions on xref:../updat
 ====
 
 [id="ocp-4-7-0-ga"]
-=== RHBA-2021:1234 - {product-title} 4.7 image release and bug fix advisory
+=== RHEA-2020:5633 - {product-title} 4.7.0 image release, bug fix, and security update advisory
 
-Issued: 2021-xx-xx
+Issued: 2021-02-24
 
-{product-title} release 4.7 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2020:1234[RHBA-2020:1234] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2020:5678[RHBA-2020:5678] advisory.
+{product-title} release 4.7.0 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHEA-2020:5633[RHEA-2020:5633] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHEA-2020:5634[RHEA-2020:5634] advisory.
+
+This release also includes security updates. The list of security updates is documented in the link:https://access.redhat.com/errata/RHSA-2020:5635[RHSA-2020:5635] advisory.
 
 Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
```

Comment:
You could embed the link in the sentence as with the previous paragraphs. For example:

"Space precluded documenting all of the container images for this release in the advisory. See the link:https://access.redhat.com/solutions/5820231[{product-title} 4.7.0 container image list] for notes on the container images in this release."

Reply:
I needed to read ""Space precluded documenting" a number of times before I understood its meaning, but this might just be me. You could potentially reword that for clarity or even remove the first sentence in this final paragraph altogether.

Reply:
Point well taken. Don't want to throw L10N a curveball, but will revisit post-GA.